package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package format="2">
   <name>manif</name>
-  <version>0.0.0</version>
+  <version>0.0.3</version>
```

Comment:
Is it OK to require the version specification twice? Once in CMakeLists and once here?

Reply:
Note that it is also specified in the `setup.py` altho there I could manage to retrieve the version from the `CMakeLists.txt`. It is indeed unfortunate to specify it in several places but I couldn't find a workaround. Note that they are used for the same purpose by two different tools.
`CMake` requires the version in the `CMakeLists.txt` when invoking e.g. `find_package(manif 0.0.3)` and similarly ROS requires it in the `package.xml` for the same versioning purpose

Reply:
yeah ok I see. No problem. Approving